cnf-tests/.konflux/Dockerfile

```diff
@@ -65,13 +65,13 @@ COPY --from=builder-sctptester /sctptest /usr/bin/sctptest
 COPY --from=builder-hugepages-allocator /hugepages-allocator /usr/bin/hugepages-allocator
 
 RUN sed -i 's/quay.io\/openshift-kni\//registry.redhat.io\/openshift4\//g' /usr/local/etc/cnf/images.json
-RUN sed -i 's/cnf-tests:4.19/cnf-tests-rhel9:v4.19/g' /usr/local/etc/cnf/images.json
-RUN sed -i 's/dpdk:4.19/dpdk-base-rhel8:v4.19/g' /usr/local/etc/cnf/images.json
+RUN sed -i 's/cnf-tests:4.20/cnf-tests-rhel9:v4.20/g' /usr/local/etc/cnf/images.json
+RUN sed -i 's/dpdk:4.20/dpdk-base-rhel9:v4.20/g' /usr/local/etc/cnf/images.json
 
-ENV OCP_VERSION=4.19
+ENV OCP_VERSION=4.20
 ENV IMAGE_REGISTRY=registry.redhat.io/openshift4/
 ENV CNF_TESTS_IMAGE=cnf-tests-rhel9:v${OCP_VERSION}
-ENV DPDK_TESTS_IMAGE=dpdk-base-rhel8:v${OCP_VERSION}
+ENV DPDK_TESTS_IMAGE=dpdk-base-rhel9:v${OCP_VERSION}
```

Comment:
I think we need to fix this in 4.19 as well no ?

Reply:
correct. 4.19 configuration will anyhow need to be adjusted. will have branch-targettted PRs once the 4.20 is complete e2e.